### DISCUSSION TITLE
Multiple shareable links for the same album

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#926] [Enhancements]
There should be an option to create multiple links for the same album, each with it's own identifier (name / description) where one link could be password protected, other could allow for uploads, ...

Currently a workaround is to create multiple albums with the same items, and generate a link per album. Allowing for multiple links will simplify this experience.

**Comment by Rubyer77** [comment]
@vishnukvmd I want that capability too. Do you know of any service that has that capability? No such option on Dropbox, Box, Flickr...
